apps/designer/app/designer/features/style-panel/controls/text/text-control.tsx

```diff
@@ -29,10 +29,10 @@ export const TextControl = ({
   >();
 
   return (
-    <Tooltip
+    <EnhancedTooltip
       content={styleConfig.label}
-      delayDuration={600}
-      disableHoverableContent={true}
+      delayDuration={1600}
+      disableHoverableContent={false}
```

Comment:
I think you don't want to set this up for each tooltip individually

Reply:
Yep. If `skipDelayDuration = 0` would work as expected.